service/index.js

```diff
@@ -119,7 +124,7 @@ app.get(/^\/v([12])\/polyfill(\.\w+)(\.\w+)?/, function(req, res) {
 	var apiVersion = parseInt(req.params[0], 10);
 	var firstParameter = req.params[1].toLowerCase();
 	var minified =  firstParameter === '.min';
-	var fileExtension = minified ? req.params[2].toLowerCase() : firstParameter;
+	var fileExtension = (minified && req.params[2]) ? req.params[2].toLowerCase() : firstParameter;
```

Comment:
do we even need `minified` here?  The check for `req.params[2]` seems to do the job.